Prog/average_unitsize_sf_mf.sas

```diff
@@ -0,0 +1,59 @@
+/**************************************************************************
+ Program:  average_unitsize_sf_mf.sas
+ Library:  RegHsg
+ Project:  NeighborhoodInfo DC
+ Author:   Yipeng Su
+ Created:  2/6/19
+ Version:  SAS 9.2
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  tabulate sf and mf average unit size for DC MSA
+ codebook: https://www.census.gov/data-tools/demo/codebook/ahs/ahsdict.html?s_appName=ahsdict&s_searchvalue=INTSTATUS
+ Modifications: 
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( RegHsg)
+
+/*
+%include "F:\HouseFin\AHS\code\pets\Apply AHS Formats_2017.sas";
+%include "F:\HouseFin\AHS\code\pets\Apply AHS 2013 Formats.sas";
+libname ahs "K:\Housefin3\Monthly\AHS";
+*/
+
+data test;
+set RegHsg.ahs2017m;
+keep OMB13CBSA;
+run;
+
+proc freq data=test;
+tables OMB13CBSA;
```

Comment:
i'm not seeing "47900"?

Reply:
yes, 47900 dont exist in metropolitan dataset so you'll notice I used the national data in the actual calculation. @lhendey